src/mapper/pkg/resolvers/schema.helpers.resolvers.go

```diff
@@ -314,6 +315,33 @@ func (r *Resolver) handleAzureOperationReport(ctx context.Context, operation mod
 	return nil
 }
 
+func (r *Resolver) handleTrafficLevelReport(ctx context.Context, results model.TrafficLevelResults) error {
+	for _, report := range results.Results {
+		sourceIdentity, err := r.resolveIPToIdentity(ctx, report.SrcIP)
+
+		if err != nil {
+			logrus.WithError(err).Errorf("Could not resolve source IP %s to identity", report.SrcIP)
```

Comment:
use logrus.WithField for templated fields:
```suggestion
			logrus.WithError(err).WithFields("srcIP", report.SrcIP).Error("Could not resolve source IP to identity")
```